tests/integration/test_pwm_service.py

```diff
@@ -119,12 +119,12 @@ def test_get_enabled(pwm_service, pwm_num):
     assert isinstance(enabled, bool)
 
 def test_with_edgepi(pwm_service):
-    """Manual testing of PWM functionality."""
-    pwm_service.init_pwm(pwm_num=PWMPins.PWM1)
-    pwm_service.set_config(pwm_num=PWMPins.PWM1, duty_cycle=0)
-    pwm_service.enable(PWMPins.PWM1)
+    """Used for manual testing of PWM functionality."""
+    pwm_service.init_pwm(pwm_num=PWMPins.PWM2)
+    pwm_service.set_config(pwm_num=PWMPins.PWM2, polarity=Polarity.NORMAL)
+    pwm_service.enable(PWMPins.PWM2)
     time.sleep(3)
-    pwm_service.set_config(pwm_num=PWMPins.PWM1, duty_cycle=0.5)
+    pwm_service.set_config(pwm_num=PWMPins.PWM2, polarity=Polarity.INVERSED)
     time.sleep(3)
-    pwm_service.set_config(pwm_num=PWMPins.PWM1, duty_cycle=1)
+    pwm_service.set_config(pwm_num=PWMPins.PWM2, polarity=Polarity.NORMAL)
```

Comment:
are there any return values from those services that we can assert on?

Reply:
Yes, but but I already did that for all these functions in the previous integration tests. This one was to just manually see on the EdgePi if it worked. For example, I used a fan connected to the EdgePi to see the changes.